mypy/fastparse.py

```diff
@@ -294,11 +294,50 @@ def translate_expr_list(self, l: Sequence[AST]) -> List[Expression]:
             res.append(exp)
         return res
 
-    def translate_stmt_list(self, l: Sequence[AST]) -> List[Statement]:
+    def get_line(self, n: ast3.stmt) -> int:
+        if (isinstance(n, (ast3.AsyncFunctionDef, ast3.ClassDef, ast3.FunctionDef))
+                and n.decorator_list):
+            return n.decorator_list[0].lineno
+        return n.lineno
+
+    def translate_stmt_list(self,
+                            l: Sequence[ast3.stmt],
+                            module: bool = False) -> List[Statement]:
+
```

Comment:
I'd leave this blank line out, as well a the one below this block comment. (Ditto for other block comments below; in general you sprinkle more whitespace throughout the code than is mypy's tradition.)

Reply:
I've removed a lot of my extra whitespace throughout.